source/configuration/modules/omfwd.rst

```diff
@@ -1,385 +1,540 @@
+**************************************
 omfwd: syslog Forwarding Output Module
-======================================
+**************************************
 
-**Module Name:**  **omfwd**
+===========================  ===========================================================================
+**Module Name:**             **omfwd**
+**Author:**                  `Rainer Gerhards <http://rainer.gerhards.net/>`_ <rgerhards@adiscon.com>
+===========================  ===========================================================================
 
-**Author:**       Rainer Gerhards <rgerhards@adiscon.com>
+
+Purpose
+=======
 
 The omfwd plug-in provides the core functionality of traditional message
 forwarding via UDP and plain TCP. It is a built-in module that does not
 need to be loaded.
 
  
-**Note: this documentation describes features present in v7+ of
-rsyslog. If you use an older version, scroll down to "legacy
-parameters".** If you prefer, you can also `obtain a specific version
-of the rsyslog
-documentation <http://www.rsyslog.com/how-to-obtain-a-specific-doc-version/>`_.
+Notable Features
+================
 
- 
+
+Configuration Parameters
+========================
+
+.. note::
+
+   Parameter names are case-insensitive.
 
 Module Parameters
 -----------------
 
-Note: parameter names are case-insensitive.
+Template
+^^^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
 
--  **Template** [templateName]
+   "word", "RSYSLOG_TraditionalForwardFormat", "no", "``$ActionForwardDefaultTemplateName``"
 
-   sets a non-standard default template for this module.
+Sets a non-standard default template for this module.
  
 
 Action Parameters
 -----------------
 
-Note: parameter names are case-insensitive.
+Target
+^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "none", "no", "none"
+
+Name or IP-Address of the system that shall receive messages. Any
+resolvable name is fine.
+
+
+Port
+^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "514", "no", "none"
+
+Name or numerical value of port to use when connecting to target.
+
+
+Protocol
+^^^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "udp", "no", "none"
+
+Type of protocol to use for forwarding. Note that \`\`tcp'' means
+both legacy plain tcp syslog as well as RFC5425-based TCL-encrypted
+syslog. Which one is selected depends on the protocol drivers set
+before the action commend. Note that as of 6.3.6, there is no way to
```

Comment:
> Which one is selected **depends on the protocol drivers set before the action commend**. 

Looks like a mispelling and possibly a reference to the [`obsolete legacy`](http://www.rsyslog.com/doc/v8-stable/configuration/conf_formats.html) configuration format having specific directive ordering requirements.

> Note that as of 6.3.6, there is no way to specify this within the action itself.

Does the [`advanced`](http://www.rsyslog.com/doc/v8-stable/configuration/conf_formats.html) configuration format provide that functionality now?

Reply:
I have changed the description for that parameter. Yes it does with the StreamDriver parameter.